packages/react-components/src/organisms/UserProfileBackground.tsx

```diff
@@ -79,10 +74,10 @@ const UserProfileBackground: React.FC<UserProfileBackgroundProps> = ({
             <p>{role}</p>
           </div>
         </div>
-        {!!labsList.length && (
+        {!!labs.length && (
           <div css={detailsContentStyle}>
             <Headline3 styleAsHeading={5}>Labs</Headline3>
-            {labsList}
+            <UserProfileLabText labs={labs} />
```

Comment:
Not sure about this. You are now putting in an elipsis when we were not previously.
```suggestion
{getUniqueCommaStringWithSuffix(labs.map(({name}) => name), 'Lab',)}
```